src/main/java/org/broadinstitute/hellbender/engine/ReferenceFileSource.java

```diff
@@ -35,8 +48,7 @@
      * @param fastaFile reference fasta file
      */
     public ReferenceFileSource(final File fastaFile) {
```

Comment:
I think we can just get rid of this method completely and switch to path in the tests that use it.

Reply:
Agreed, and done.